backend/src/apiserver/client/minio.go

```diff
@@ -16,31 +16,61 @@ package client
 
 import (
 	"fmt"
+	"net/http"
 	"time"
 
 	"github.com/cenkalti/backoff"
 	"github.com/golang/glog"
 	minio "github.com/minio/minio-go"
+	credentials "github.com/minio/minio-go/pkg/credentials"
 	"github.com/pkg/errors"
 )
 
+func getEndpoint(host, port string) string {
+	if port == "" {
+		return host
+	}
+	return fmt.Sprintf("%s:%s", host, port)
+}
+
+// createCredentialProvidersChain creates a chained providers credential for a minio client
+func createCredentialProvidersChain(endpoint, accessKey, secretKey string) *credentials.Credentials {
```

Comment:
seems endpoint is not being used. could we clean it up?

Reply:
Any reason this wasn't addressed? Given it's a new contribution I wouldn't expect to find such cruft.